### PR TITLE
Add ServiceFabric schema version 2017-07-01-preview

### PR DESCRIPTION
**Pull request by juhacket**

### CODE REVIEW

**Comment by juhacket** [comment]
@Tiano2017 - please review and merge this change as it is causing live site incidents for our team.

File: tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+            },
+            {
+              "message": "Invalid type: object (expected string)",
+              "dataPath": "/properties",
+              "schemaPath": "/properties/properties/oneOf/1/type"
+            }
+          ]
+        }
+      ],
+<<<<<<< HEAD
```
Comment (TianoMS) [inline tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json]: please fix.

Comment (juhacket) [reply tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json]: Fixed

Comment (juhacket) [reply tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json]: @TianoMS - please review.

File: tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            },
+            {
+              "message": "Invalid type: object (expected string)",
+              "dataPath": "/properties",
+              "schemaPath": "/properties/properties/oneOf/1/type"
+            }
+          ]
+        }
+      ],
+      "definition": "https://schema.management.azure.com/schemas/2016-09-01/Microsoft.ServiceFabric.json#/resourceDefinitions/clusters",
```
Comment (TianoMS) [inline tests/2017-07-01-preview/Microsoft.ServiceFabric.tests.json]: >2016-09-01 [](start = 65, length = 10)

should be 2017-07-01-preview.

**Pull request merged**